src/NetworkBinding.cpp

```diff
@@ -301,8 +285,10 @@ void NetworkBinding::Init(v8::Local<v8::Object> exports, v8::Local<v8::Object> m
     Nan::SetPrototypeMethod(tpl, "shutdown", Shutdown);
     Nan::SetPrototypeMethod(tpl, "hash", Hash);
 
-    constructor.Reset(tpl->GetFunction());
-    module->Set(Nan::New("exports").ToLocalChecked(), tpl->GetFunction());
+    constructor.Reset();
```

Comment:
I'm confused what this is supposed to do, but removing a param seems incorrect...